### DISCUSSION TITLE
Provider Version and Github Enterprise Server Compatibility

### DISCUSSION THREAD

**Discussion by Danielku15** [integrations/terraform-provider-github#1870] [Q&A]
Is there somewhere a table where we can lookup which provider version is compatible with which GitHub Enterprise Server? We use the latest provider but get some API errors in our GitHub Enterprise Server 3.8.4. 

If there is no table, could it be taken up to add one to the readme?

**Comment by shansrii** [comment]
Hi @Danielku15 
 I don't see an answer to your question here but probably you figured it on your own by now.
 Do you have the list now ? Could you please share it

**Reply by Danielku15** [reply]
Sorry, I have no list.
